### ISSUE TITLE
Panic on opening some DTED files

### ISSUE THREAD

**Issue by Seb105** [fizyk20/dted#3]
I recently came across some DTED files that use `NA  ` as the none value for innacuracy, whereas currently only `NA$$` is accepted.
This causes a panic on opening the file.

**Comment by arpadav** [comment]
@Seb105 i recently was running into similar issues, and i needed more functionality. I forked this and changed some stuff around, planning on adding more functionality later. See: https://crates.io/crates/dted2 

let me know if your issue is resolved with this one, i should have accounted for it in the parser

**Comment by Seb105** [comment]
Yeah, the new library works. Rewrite to convert wasn't too bad either.
